src/main/java/uk/gov/hmcts/ccd/v2/internal/controller/UICaseController.java

```diff
@@ -117,4 +124,42 @@ public UICaseController(
 
         return ResponseEntity.ok(new UIEventViewResource(caseHistoryView, caseId));
     }
+
+    @GetMapping(
+        path = "/{caseId}/events",
```

Comment:
So this looks like internal endpoint but the story talks about external which would be located in equivalent CaseController class.

Reply:
Changed to external